perfkitbenchmarker/providers/aws/aws_emr.py

```diff
@@ -61,19 +62,35 @@ def __init__(self, spark_service_spec):
       self.machine_type = DEFAULT_MACHINE_TYPE
     self.cmd_prefix = util.AWS_PREFIX + ['emr']
     if FLAGS.zones:
-      region = util.GetRegionFromZone(FLAGS.zones[0])
-      self.cmd_prefix += ['--region', region]
+      self.region = util.GetRegionFromZone(FLAGS.zones[0])
+    else:
+      cmd = ['aws', 'configure', 'get', 'region']
+      stdout, _, _ = vm_util.IssueCommand(cmd)
+      self.region = stdout.strip()
+    self.cmd_prefix += ['--region', self.region]
+    self.bucket_to_delete = None
+
+  def _CreateLogBucket(self):
+    bucket_name = 's3://pkb-' + FLAGS.run_uri + '-emr-' + self.region
```

Comment:
I think the style guide prefers a format or % in cases like these https://google.github.io/styleguide/pyguide.html?showone=Strings#Strings